src/hotspot/share/utilities/nativeCallStack.cpp

```diff
@@ -113,6 +113,7 @@ void NativeCallStack::print_on(outputStream* out) const {
   DEBUG_ONLY(assert_not_fake();)
   for (int i = 0; i < NMT_TrackingStackDepth && _stack[i] != nullptr; i++) {
     print_frame(out, _stack[i]);
+    out->cr();
   }
   out->cr();
```

Comment:
Do you need the `cr()` after the loop now? Guess it is a matter of personal preference depending on the context in which this gets printed.

Reply:
Good point. Let's restore to pre [JDK-8333994](https://bugs.openjdk.org/browse/JDK-8333994) behavior, where it does not have `cr()` after the loop.